setup/create_windows_msi.yml

```
@@ -0,0 +1,72 @@
+parameters:
+  working_dir: ''
```

```suggestion
    - name: working_dir
      type: string
```

nit. Similar changes for the other params,